chia/wallet/wallet_action_store.py

```diff
@@ -37,11 +37,11 @@ async def create(cls, db_wrapper: DBWrapper):
             )
         )
 
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS name on action_queue(name)")
+        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS aq_name on action_queue(name)")
 
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS wallet_id on action_queue(wallet_id)")
+        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS aq_wallet_id on action_queue(wallet_id)")
 
-        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS wallet_type on action_queue(wallet_type)")
+        await self.db_connection.execute("CREATE INDEX IF NOT EXISTS aq_wallet_type on action_queue(wallet_type)")
```

Comment:
I expect a strong rationale for all of these name changes. Without dropping the previous indices, you also make the database larger (for not apparent reason)

Reply:
same reason. Index named "wallet_type" was already created on table coin_record.
I'd recommend to use table acronym in index name or something similar so such problems don't happen.